app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/settings/SettingsExampleFragment.java

```diff
@@ -30,9 +30,10 @@
 import java.util.Arrays;
 import java.util.Stack;
 
-public class SettingsExampleFragment extends LeanbackSettingsFragment implements DialogPreference.TargetFragment {
+public class SettingsExampleFragment extends LeanbackSettingsFragment
+        implements DialogPreference.TargetFragment {
 
-    private final Stack<Fragment> fragments = new Stack<Fragment>();
+    private static final Stack<Fragment> fragments = new Stack<Fragment>();
```

Comment:
Actually our settings example activity has a bug (the app will be crashed once this activity is started). I will fix this bug in an independent CL, some change was put in this CL accidentally, will be more careful next time.